compiler/luci/pass/src/PropagateQParamBackwardPass.cpp

```diff
@@ -466,6 +466,9 @@ bool PropagateQParamBackwardPass::run(loco::Graph *g)
     auto circle_node = loco::must_cast<luci::CircleNode *>(node);
     INFO(l) << "PropagateQParamBackwardPass visit node: " << circle_node->name() << std::endl;
 
+    if (circle_node->quantparam() == nullptr)
+      continue;
```

Comment:
This code is necessary as Bool/S32/S64 type Ops do not have quantparam at this point.

Reply:
(optional) maybe add a NOTE or TODO about this?